src/libraries/Common/tests/System/Security/Cryptography/PlatformSupport.cs

```diff
@@ -17,5 +17,7 @@ internal static class PlatformSupport
 
         // Whether or not the current platform supports RC2
         internal static readonly bool IsRC2Supported = !PlatformDetection.IsAndroid;
+
+        internal static readonly bool IsAndroidVersionAtLeast31 = OperatingSystem.IsAndroidVersionAtLeast(31);
```

Comment:
Looks like this needs to be in something like `#if NET_5_OR_GREATER`, with the else saying false.

Reply:
yup fixed